backend/lcfs/db/models/notification/NotificationType.py

```diff
@@ -1,11 +1,13 @@
 import enum
 from lcfs.db.base import BaseModel, Auditable
-from sqlalchemy import Column, Integer, Enum, Text
+from sqlalchemy import Column, Integer, Text, String
 from sqlalchemy.orm import relationship
 
 
 class NotificationTypeEnum(enum.Enum):
```

Comment:
Since we no longer using the NotificationTypeEnum, can we remove this?

Reply:
Of course! I also needed to update the email classes as they depend on that. Hopefully, I don't break anything!